r/rethinkdb/rethinkdb_ubi_8.5.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+# -----------------------------------------------------------------------------
+#
+# Package       : rethinkdb
+# Version       : v2.4.x
+# Source repo   : https://github.com/rethinkdb/rethinkdb.git
+# Tested on     : UBI 8.5
+# Language      : C++
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Vinod K <Vinod.K1@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+
+PACKAGE_VERSION=v2.4.x
```

Comment:
@vinodk99 - Can you make change to accept dynamic PACKAGE_VERSION  value with $1 parameter to script `PACKAGE_VERSION=${1:-v2.4.x}`